apps/www/src/lib/registry/new-york/block/Dashboard05.vue

```diff
@@ -0,0 +1,772 @@
+<script lang="ts">
+export const description = 'A order tracking page. The order page has a sidebar navigation and a main content area.'
+export const iframeHeight = '780px'
+export const containerClass = 'w-full h-full'
+</script>
+
+<script setup lang="ts">
+import {
+  CircleUser,
+  Copy,
+  CreditCard,
+  File,
+  Home,
+  LineChart,
+  ListFilter,
+  MoreVertical,
+  Package,
+  Package2,
+  PanelLeft,
+  Search,
+  Settings,
+  ShoppingCart,
+  Truck,
+  Users2,
+} from 'lucide-vue-next'
```

Comment:
You should use `@radix-icons/vue` for new-york style

Reply:
All **new-york** dashboards use icons from `'lucide-vue-next'`, should i change them too?

Reply:
@sadeghbarati It's weird that the original repo is using `lucide` for `new-york` style too. I find it so inconsistent 😅

Reply:
Let's keep the icon same for now ya @LucasBois1  @sadeghbarati 😁
If `ui.shadcn` updates it in the future, we can do the same too